src/containers/MealsPage/MealsPage.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import RecipeCard from '../../components/Cards/Recipe-Card/Recipe-Card';
```

Comment:
I believe this import is wrong, it should be like this
```suggestion
import RecipeCard from '../../components/Card/RecipeCard/RecipeCard';
```